samples/Azure.ResourceManager.Sample/Generated/Models/SubscriptionResourceGetVirtualMachineImagesOptions.cs

```diff
@@ -40,12 +40,10 @@ public SubscriptionResourceGetVirtualMachineImagesOptions(AzureLocation location
         /// <summary> A valid image SKU. </summary>
         public string Skus { get; }
         /// <summary> The expand expression to apply on the operation. </summary>
-        public string Expand { get; set; } = null;
```

Comment:
These are removed on purpose because they are meaningless.

For collections, this initialization to `null` never worked.
For non-collection types, this initialization is pointless because they will always be initialized as default values when not explicitly assigned